applications/StructuralMechanicsApplication/tests/InitialStateElasticity/initial_strain_shell_Q4_thick_test_parameters.json

```diff
@@ -0,0 +1,156 @@
+        "max_iteration"                   : 10,
+        "rotation_dofs"                   : true,
+        "volumetric_strain_dofs"          : false
+    },
+    "processes"        : {
+        "constraints_process_list" : [{
+            "python_module" : "assign_vector_variable_process",
+            "kratos_module" : "KratosMultiphysics",
+            "process_name"  : "AssignVectorVariableProcess",
+            "Parameters"    : {
+                "model_part_name" : "Structure.DISPLACEMENT_Vertex_LB",
+                "variable_name"   : "DISPLACEMENT",
+                "interval"        : [0.0,"End"],
+                "constrained"     : [true,true,true],
+                "value"           : [0.0,0.0,0.0]
+            }
+        },{
+            "python_module" : "assign_vector_variable_process",
+            "kratos_module" : "KratosMultiphysics",
+            "process_name"  : "AssignVectorVariableProcess",
+            "Parameters"    : {
+                "model_part_name" : "Structure.DISPLACEMENT_Vertex_LT",
+                "variable_name"   : "DISPLACEMENT",
+                "interval"        : [0.0,"End"],
+                "constrained"     : [false,false,true],
+                "value"           : [null,null,0.0]
+            }
+        },{
+            "python_module" : "assign_vector_variable_process",
+            "kratos_module" : "KratosMultiphysics",
+            "process_name"  : "AssignVectorVariableProcess",
+            "Parameters"    : {
+                "model_part_name" : "Structure.DISPLACEMENT_Vertex_RB",
+                "variable_name"   : "DISPLACEMENT",
+                "interval"        : [0.0,"End"],
+                "constrained"     : [false,true,true],
+                "value"           : [null,0.0,0.0]
+            }
+        },{
+            "python_module" : "assign_vector_variable_process",
+            "kratos_module" : "KratosMultiphysics",
+            "process_name"  : "AssignVectorVariableProcess",
+            "Parameters"    : {
+                "model_part_name" : "Structure.DISPLACEMENT_Vertex_RT",
+                "variable_name"   : "DISPLACEMENT",
+                "interval"        : [0.0,"End"],
+                "constrained"     : [false,false,true],
+                "value"           : [null,null,0.0]
+            }
+        }],
+        "loads_process_list"       : [{
+            "python_module" : "set_initial_state_process",
+            "kratos_module" : "KratosMultiphysics",
+            "process_name"  : "set_initial_state_process",
+            "Parameters"    : {
+            "mesh_id"         : 0,
+            "model_part_name" : "Structure.Parts_Shell_Slab",
+            "dimension"       : 2,
+            "imposed_strain"  : [1.0e-4,-5.0e-5,-2e-4],
+            "imposed_stress"  : [0,0,0],
+            "imposed_deformation_gradient" : [[1,0,0],[0,1,0],[0,0,1]],
+            "interval"        : [0.0,"End"]
+            }
+        }],
+        "_print_output_process" : [{
```

Comment:
the underscore syntax is not reliable (in terms of it works in some cases and not in others) and is from a time before we could have comments in the json

Hence I would comment this block. Commenting works same as in C++

What happens here is that this process still gets executed and hence renders the test useless as it constantly writes new reference results